packages/shared/src/badges/Badge.scss

```
@@ -1,10 +1,10 @@
 .ocs-preview-badge {
   &.pf-c-label {
     --pf-c-label--BackgroundColor: #d93f00;
-    --pf-c-label--BorderRadius: var(--pf-global--BorderRadius--sm);
+    --pf-c-label--BorderRadius: var(--pf-v5-global--BorderRadius--sm);
```

aren't these variable names gonna get changed too ??
```suggestion
    --pf-v5-c-label--BorderRadius: var(--pf-v5-global--BorderRadius--sm);
```